README.md

```diff
@@ -147,50 +147,57 @@ If `beforeAll` and `afterAll` are called multiple times, the callbacks are execu
-      --reporter, -r       Output additional report format. This option can be used
-                           multiple times to add multiple reporters. Options:
-                           junit, nyan, dot, markdown, html.
-                                                                       [default: []]
-      --output, -o         Specifies output file when using additional file-based
-                           reporter. This option can be used multiple times if
-                           multiple file-based reporters are used.
-                                                                       [default: []]
-      --header, -h         Extra header to include in every request. This option
-                           can be used multiple times to add multiple headers.
-                                                                       [default: []]
-      --sorted, -s         Sorts requests in a sensible way so that objects are not
-                           modified before they are created. Order: CONNECT,
-                           OPTIONS, POST, GET, HEAD, PUT, PATCH, DELETE, TRACE.
-                                                                    [default: false]
-      --user, -u           Basic Auth credentials in the form username:password.
-                                                                     [default: null]
-      --inline-errors, -e  Determines whether failures and errors are displayed as
-                           they occur (true) or agregated and displayed at the end
-                           (false).
-                                                                    [default: false]
-      --details, -d        Determines whether request/response details are included
-                           in passing validations.
-                                                                    [default: false]
-      --method, -m         Restrict tests to a particular HTTP method (GET, PUT,
-                           POST, DELETE, PATCH). This option can be used multiple
-                           times to allow multiple methods.
-                                                                       [default: []]
-      --color, -c          Determines whether console output should include colors.
-                                                                     [default: true]
-      --level, -l          The level of logging to output. Options: silly, debug,
-                           verbose, info, warn, error.
-                                                                   [default: "info"]
-      --timestamp, -t      Determines whether console output should include
-                           timestamps.
-                                                                    [default: false]
-      --help               Show usage information.
-      --version            Show version number.
+  --dry-run, -y        Do not run any real HTTP transaction, only parse
```

Comment:
It looks like this isn't indented enough to be considered a code block by markdown (you can see in the preview for the file).